icu4c/source/i18n/units_complexconverter.h

```diff
@@ -48,17 +48,33 @@ namespace units {
  */
 class U_I18N_API ComplexUnitsConverter : public UMemory {
   public:
+    /**
+     * Constructs `ComplexUnitsConverter` for an `targetUnit` that could be Single, Compound or Mixed.
+     * In case of:
+     * 1- Single and Compound units,
+     * the conversion will not perform anything, the input will be equal to the output.
+     * 2- Mixed Unit
+     * the conversion will consider the input in the biggest unit. and will convert it to be spread throw
```

Comment:
"to be spread throw the input units" -> throw is the wrong word (through), and it's represented in the mixed units specified by targetUnit, not anything with "input units".